styles/main-style.css

```diff
@@ -34,32 +34,43 @@ body{
     font-family: var(--general-font);
     display: flex;
     flex-direction: column;
-    row-gap: 10px;
```

Comment:
remove the commented code if not necessary.

Reply:
Will remove the commented part.

Reply:
Vaibhav, I am not sure but maybe you have to resolve the conservation.